### PR TITLE
[W5.7][F11-3] Ng Peng Nam, Sean

### PR DESCRIPTION
**Pull request by pengnam**
Added case-insensitive support for the find function
Updated tests to reflect modified feature

### CODE REVIEW

**Comment by jeffkwoh** [comment]
Great feature, wish I thought of this.  Everything looks great, just a friendly reminder to update the user guide to reflect the change in behaviour of find.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -47,14 +47,26 @@ public CommandResult execute() {
...
+            lowercaseKeywords.add(keyword.toLowerCase());
+        }
+
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            //Convert words in person's name to lowercase
+            Set<String> wordsInName = new HashSet<>();
+            for (String word:person.getName().getWordsInName()) {
```
Comment (jelneo) [inline src/seedu/addressbook/commands/FindCommand.java:60]: Would be good to follow the format above in line 57 (`for (ReadOnlyPerson person : addressBook.getAllPersons())`). where there are spaces before and after `:`

**Review by jelneo — APPROVED** [review]
It is encouraged to update the user guide. Please close this PR after reading my comments

File: src/seedu/addressbook/commands/FindCommand.java (hunk 1 above)
Comment (pengnam) [reply src/seedu/addressbook/commands/FindCommand.java:60]: Thanks for pointing it out!

**Pull request closed without merging**